### PR TITLE
Create licenses.md

### PR DESCRIPTION
**Pull request by beckettsean**
partial fix of #281

### CODE REVIEW

File: content/docs/v0.9/about/licenses.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+title: Licenses
+---
+
+## InfluxDB License
+
+InfluxDB uses the [MIT license](https://github.com/influxdb/influxdb/blob/master/LICENSE)
+
+## Dependency Licenses
+
+InfluxDB has dependencies that use the MIT, BSD, MPL, and WTFPL licenses. The dependencies and their licenses are listed in [LICENSE_OF_DEPENDENCIES.md](https://github.com/influxdb/influxdb/blob/master/LICENSE_OF_DEPENDENCIES.md).
```
Comment (gunnaraasen) [inline content/docs/v0.9/about/licenses.md]: This link isn't supposed to work yet, right?

Comment (beckettsean) [inline content/docs/v0.9/about/licenses.md]: yeah, it doesn't yet exist but that is the proper URL.

@mjdesa any idea when the PR for LICENSES_OF_DEPENDENCIES will get merged?

**Comment by gunnaraasen** [comment]
+1

**Comment by beckettsean** [comment]
Blocked pending https://github.com/influxdb/influxdb/pull/3891

**Pull request merged**